packages/sdk/src/libs/index.ts

```diff
@@ -6,7 +6,7 @@ export { default as Coingecko } from "./coingecko";
 export { default as Multicall } from "./multicall";
 
 // types
-import type { TypedEventFilter, TypedEvent } from "@uma/core/contract-types/ethers/commons";
+import type { TypedEventFilter, TypedEvent } from "@uma/core/types/contract-types/ethers/commons";
```

Comment:
this was also the wrong directory. took me a while to realize this

Reply:
Yeah, these import paths in core are very odd. The newer packages will fix these oddities. Sorry for the weirdness. Didn't realize using types like this would cause such strange paths, as I misunderstood typescript resolution.